File: auth-single-server/src/main/java/com/cartobucket/auth/config/JsonbConfiguration.java
package com.cartobucket.auth.config;

import jakarta.enterprise.context.ApplicationScoped;
import jakarta.enterprise.inject.Produces;
import jakarta.json.bind.Jsonb;
import jakarta.json.bind.JsonbBuilder;
import jakarta.json.bind.JsonbConfig;

@ApplicationScoped
public class JsonbConfiguration {
    
    @Produces
    @ApplicationScoped
    public Jsonb createJsonb() {
        JsonbConfig config = new JsonbConfig();
        return JsonbBuilder.create(config);
    }
}

File: auth-single-server/src/main/java/com/cartobucket/auth/config/JsonbConfiguration.kt
package com.cartobucket.auth.config

import jakarta.enterprise.context.ApplicationScoped
import jakarta.enterprise.inject.Produces
import jakarta.json.bind.Jsonb
import jakarta.json.bind.JsonbBuilder
import jakarta.json.bind.JsonbConfig

@ApplicationScoped
class JsonbConfiguration {
    @Produces
    @ApplicationScoped
    fun createJsonb(): Jsonb? {
        val config = JsonbConfig()
        return JsonbBuilder.create(config)
    }
}
